src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -2199,7 +2213,7 @@ public function testFactory()
 
     public function testFactoryCallable()
     {
-        $requestFactory = new class {
+        $requestFactory = new class() {
```

Comment:
not sure about our code style here, with `()` or without

Reply:
fabbot patched it